tailwind.config.cjs

```diff
@@ -1,14 +1,28 @@
+const colors = require('tailwindcss/colors');
+
 const config = {
   content: ["./src/**/*.{html,js,svelte,ts}"],
-
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {
+        'body': ['Inter', 'Roboto', 'Aria', 'sans-serif']
```

Comment:
When adding web fonts please extend the native font stack included instead of overriding it. You can access the default font family with an import: `const { fontFamily } = require('tailwindcss/defaultTheme');`